test/horizon/client/default_test.exs

```diff
@@ -12,21 +12,42 @@ defmodule Stellar.Horizon.Client.CannedHTTPClient do
           body :: String.t(),
           options :: list()
         ) :: {:ok, map()} | {:error, map()}
-  def request(:get, @base_url <> "/transactions/unknow_id", _headers, _body, _opts) do
-    json_error = Horizon.fixture(:not_found)
-    {:ok, 404, [], json_error}
-  end
 
   def request(:get, @base_url <> "/transactions/" <> _hash, _headers, _body, _opts) do
-    json_body = Horizon.fixture(:transaction)
+    json_body = Horizon.fixture("200")
     {:ok, 200, [], json_body}
   end
 
   def request(:post, @base_url <> "/transactions?tx=bad", _headers, _body, _opts) do
-    json_error = Horizon.fixture(:error)
+    json_error =  Horizon.fixture("400")
```

Comment:
Extra space after the `=`

Reply:
This was solved yesterday, I think you're reviewing an old version of the PR!